pkg/api/frontendsettings.go

```diff
@@ -366,6 +366,21 @@ func (hs *HTTPServer) getFSDataSources(c *models.ReqContext, enabledPlugins Enab
 	return dataSources, nil
 }
 
+func newAppDTO(plugin plugins.PluginDTO, settings pluginsettings.InfoDTO) *plugins.AppDTO {
+	app := &plugins.AppDTO{
+		ID:      plugin.ID,
+		Version: plugin.Info.Version,
+		Path:    plugin.Module,
+		Preload: plugin.Preload,
+	}
+
+	if settings.Enabled {
```

Comment:
Shouldn't `apps` only contain enabled apps to follow `datasources`?

Reply:
Hmm weird. Doesn't feel correct to preload a plugin that's not enabled? What would be the point of that?

Reply:
I agree 100% but I didn't want to change the existing behavior. Maybe something we can do in another PR if we should change it?

Reply:
@mckn 
we should not preload apps that are not enabled, must be a bug

Reply:
Will fix this in another PR.